app/services/forest_liana/permissions_checker.rb

```diff
@@ -0,0 +1,62 @@
+require 'date'
+
+module ForestLiana
+  class PermissionsChecker
+    @@permissions = nil
+    @@last_retrieve = nil
+
+    def initialize(resource, permission_name)
+      @collection_name = ForestLiana.name_for(resource)
+      @permission_name = permission_name
+
+      @expiration_in_seconds = !ENV['FOREST_PERMISSIONS_EXPIRATION_IN_SECONDS'].nil? ?
```

Comment:
```ruby
@expiration_in_seconds = ENV['FOREST_PERMISSIONS_EXPIRATION_IN_SECONDS'] || 3600
```

Reply:
This should be more a class variable than an instance variable by the way.

Reply:
Your example does not convert the env variable to int